public/app/features/alerting/unified/mocks/alertRuleApi.ts

```diff
@@ -80,3 +81,167 @@ export const namespaces: Record<string, RulerRuleGroupDTO[]> = {
   [grafanaRulerNamespace.uid]: [grafanaRulerGroup],
   [grafanaRulerNamespace2.uid]: [grafanaRulerEmptyGroup],
 };
+
+//-------------------- for alert history tests we reuse these constants --------------------
```

Comment:
i was not sure where to put these constants. We need clear examples of times to make the tests more understandable.